fitwsarah_frontend/src/views/PersonalTrainerPanel/TrainerFeedback.js

```diff
@@ -12,21 +13,57 @@ function Feedbacks() {
 
     const {
         isAuthenticated,
+        getAccessTokenSilently,
     } = useAuth0();
 
 
     const [feedbacks, setFeedbacks] = useState([]);
     const { t } = useTranslation('adminPanel');
+    const [accessToken, setAccessToken] = useState(null);
+
+        useEffect(() => {
+        if (isAuthenticated) {
+          const getAccessToken = async () => {
+            try {
+              const token = await getAccessTokenSilently({
+                audience: process.env.REACT_APP_AUTH0_AUDIENCE,
+                scope: configData.scope,
+              });
+              setAccessToken(token);
+            } catch (e) {
+              console.error(e.message);
+            }
+          };
+          getAccessToken();
+        }
+      }, [getAccessTokenSilently, isAuthenticated]);
```

Comment:
Maybe you should be using the shorter version that uses this from the Auth0Utils.js

`useEffect(() => {
        const fetchToken = async () => {
          const token = await getAccessToken();
          if (token) setAccessToken(token);
        };
  
        fetchToken();
      }, [getAccessToken]);`
      
      
Make sure to have import this:
`import { useGetAccessToken } from "../authentication/authUtils";`

And have this constance:
`const getAccessToken = useGetAccessToken();`

Reply:
sorry didnt look great before

` useEffect(() => {
        const fetchToken = async () => {
          const token = await getAccessToken();
          if (token) setAccessToken(token);
        };
  
        fetchToken();
      }, [getAccessToken]); `

Reply:
Thank you for the suggestion! I updated the file with the new function.